galera/Project1/Galera.cpp

```diff
@@ -32,3 +32,65 @@ int Galera::getTotalSallary() const
 {
     return _totalSallary;
 }
+
+bool Galera::hireEmployee(Employee first)
+{
+    bool k = true;
+    if (_employeeList.empty())
+    {
+        _employeeList.push_back(first);
+        return true;
+    }
+    auto x = _employeeList.begin();
+    for (int i = 0; i < _employeeList.size(); i++)
```

Comment:
Here you have kind of double iteration. Please, either throw away `x` and use `_employeeList[i]` inside the `for`-loop, as you did inside `fireEmployee` or use iterators in loop, that is 
```cpp
for (auto it = _employeeList.begin(); it != _employeeList.end(); ++it)
{
// code
}
```